woocommerce/woocommerce-ios
# [Mac Catalyst] 3. Remove Zendesk SupportSDK from Catalyst builds

Review/approve #5657 before this.


### Description
Zendesk SupportSDK does not support running on Mac Catalyst 😢
The previous PR prevented it from being linked for Catalyst builds, but we still tried to use it. This removes our use of SupportSDK when we're targeting the Mac. 

It doesn't try to do anything else to inform the user of what's going on here... however, Zendesk have indicated that a version of the SDK which should ship before the end of the year will support Catalyst, though they will not provide support (!) for that use case. I've tested with 5.3, the latest release in CocoaPods at the time of writing, which does not work with Catalyst, but 5.4 looks like it is pending release to the Pod repo, and may be the release which adds Catalyst support.

Suggested routes forward from here:
1. Use the next version of SupportSDK and revert this change, if it works acceptably.
2. Link the user out to support on the web when running on a Mac.



### Testing instructions
With this change, the app builds on Catalyst, but you will likely see a crash as you log in. The second run may avoid that crash, as long as you don't go to log in or switch store.

To build for Catalyst, select `My Mac` as the target (not "Designed for iPad") and ensure that your Mac has been added to the development provisioning profile.

iOS smoke testing should continue to show no changes, in particular I've checked the help and support pages. Full smoke testing of the live chat would be beneficial here.

### Screenshots
The Help & Support section looks pretty... thin on a Mac now!

![Screenshot 2021-12-10 at 17 03 49](https://user-images.githubusercontent.com/2472348/145613054-42ee290b-4a47-405f-9579-599186648884.jpg)





---
